### DISCUSSION TITLE
How do I get rid of obsolete OpenDCT Tuners/Encoders?

### DISCUSSION THREAD

**Discussion by wayner9** [OpenSageTV/Community#130] [General]
In the past I played around with OpenDCT.  The outcome of this is that I now have 22 obsolete tuners.  Many of these are like: "HDHomeRun HDHR4-2US Tuner 1040E3E1-0 on 92db3458734:9000".

How do I get rid of these?  

Do I go through sage.properties and blow out the lines that relate to all of these encoders?  

Or is there some other way of doing this?

**Comment by JustFred51** [comment]
Yes, to get rid of those tuners requires manually editing sage.properties.  The corresponding entries in opendct.properties (for Windows, in C:\ProgramData\OpenDCT\config) should also be removed.  Both SageTv and OpenDCT should be stopped when making the edits.

**Reply by JustFred51** [reply]
Looks good.  That'll take care of any OpenDCT tuners that Sage had been reporting.

But Sage might still re-discover non-cablecard HDHR tuners (such as the HDHR4-2US mentioned above by wayner9).

To go one step further to prevent non-cablecard HDHR tuners from again showing up in Sage's 'Add New Source', launch the HDHomeRun Setup program.  Select the Tuners tab, locate the (un)desired tuner(s) and set Source Type to Disabled and then Apply.  But personally, I avoid disabling tuners in HDHRSetup. It's just one more thing that I later forget to tweak when I want to re-add that tuner to Sage.

**Reply by JustFred51** [reply]
My comments below are in the context of Windows. I don't know anything about how HDHR works with Linux.  That being said...

I'm not exactly clear about the statement "no point in using the HDHomeRun Setup program".  It **is** needed for the HDHomerun Service to allow Sage to *initially discover* HDHR OTA tuners. I don't know whether OpenDCT will *discover* Prime tuners when the HDHR Service isn't running or HDHR Setup has set them to Disabled; never tested that.  But testing has confirmed that after Sage (and OpenDCT) has discovered & configured the OTA/Prime tuners, the HDHR Service doesn't seem to play a role in streaming data.
